bundles/org.openhab.binding.amberelectric/src/main/java/org/openhab/binding/amberelectric/internal/api/Sites.java

```diff
@@ -0,0 +1,60 @@
+/**
+ * Copyright (c) 2010-2024 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.amberelectric.internal.api;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.google.gson.JsonArray;
+import com.google.gson.JsonObject;
+import com.google.gson.JsonParser;
+
+/**
+ * Class for holding the set of parameters used to read the controller variables.
+ *
+ * @author Paul Smedley - Initial Contribution
+ *
+ */
+@NonNullByDefault
+public class Sites {
+    private static Logger LOGGER = LoggerFactory.getLogger(Sites.class);
+
+    public String siteid = "";
+    public String nmi = "";
+
+    private Sites() {
+    }
+
+    public static Sites parse(String response, String nem) {
+        LOGGER.debug("Parsing string: \"{}\"", response);
```

Comment:
Static logger will generate a SAT warning, perhaps the caller could log this response, since the logger is not used for anything else in this static method?

Reply:
Done. Also dropped this back to trace rather than debug.